File: chatwoot/python/src/AddNewCategoryToAccountExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
)

with ApiClient(configuration) as api_client:
    category_create_update_payload = models.CategoryCreateUpdatePayload(
        locale="en/es",
    )

    try:
        response = api.HelpCenterApi(api_client).add_new_category_to_account(
            account_id=0,
            portal_id=0,
            data=category_create_update_payload,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling HelpCenterApi#add_new_category_to_account: %s\n" % e)

File: chatwoot/ruby/src/AddNewCategoryToAccountExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
end

category_create_update_payload = ChatwootClient::CategoryCreateUpdatePayload.new
category_create_update_payload.locale = "en/es"

begin
    response = ChatwootClient::HelpCenterApi.new.add_new_category_to_account(
        0, # account_id
        0, # portal_id
        category_create_update_payload, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling HelpCenterApi#add_new_category_to_account: #{e}"
end
